cmd/operator-sdk/generate/csv.go

```diff
@@ -27,17 +27,19 @@ import (
 )
 
 type csvCmd struct {
-	csvVersion     string
-	csvChannel     string
-	fromVersion    string
-	operatorName   string
-	outputDir      string
-	deployDir      string
-	apisDir        string
-	crdDir         string
-	updateCRDs     bool
-	defaultChannel bool
-	makeManifests  bool
+	csvVersion       string
+	csvChannel       string
+	fromVersion      string
+	operatorName     string
+	outputDir        string
+	deployDir        string
+	apisDir          string
+	crdDir           string
+	updateCRDs       bool
+	defaultChannel   bool
+	makeManifests    bool
+	interactive      bool
+	interactivelevel gencatalog.InteractiveLevel
```

Comment:
move the line `interactivelevel gencatalog.InteractiveLevel` to top of the bool's to solve the maligne lint issue